lib/node_modules/@stdlib/ndarray/pop/README.md

```diff
@@ -0,0 +1,148 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2025 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# pop
+
+> Return an array containing a read-only truncated view of an input [`ndarray`][@stdlib/ndarray/ctor] and a read-only view of the last element(s) along a specified dimension.
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var pop = require( '@stdlib/ndarray/pop' );
+```
+
+#### pop( x, dim )
```

Comment:
Should we default to `dim = -1` here and make `dim` an optional argument?

Reply:
This would make `pop( vector )` a more direct analog to `Array.prototype.pop`, rather than needing to do `pop( vector, 0 )`, which is more verbose when `dim` can only be `0` (or `-1`).